### ISSUE TITLE
Error in "code-block" directive:      maximum 1 argument(s) allowed, 2 supplied.

### ISSUE THREAD

**Issue by uralbash** [myint/rstcheck#5]
Hello, I'm trying to set the arguments in the directives, but get an error:

```python
.. code-block:: python
    :linenos:

    print("Hello!")
```

```bash

$ rstcheck source/hello.rst
source/hello.rst:87: (ERROR/3) Error in "code-block" directive:     
maximum 1 argument(s) allowed, 2 supplied.                                                      
                                                                                                
.. code-block:: python
    :linenos:

    print("Hello!")                                                     
```

Without `:linenos:` work OK

**Comment by myint** [comment]
Is `:linenos:` some Sphinx-specific thing? If so, please see #1, #2, #3, and #4.

**Comment by uralbash** [comment]
There is an idea to include it directly from sphinx

**Comment by myint** [comment]
This is fixed in #6.
